git_machete/short_docs.py

```diff
@@ -0,0 +1,31 @@
+from typing import Dict
+
```

Comment:
Actually, would be nice if (at least for the time of review), `short_docs.py` and `long_docs.py` are located into a single file, called just `docs.py` as before this change... this way we'll clearly see what's changed!

Reply:
(resolved?)